circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/annotation/BeekeeperHiveTableAnnotator.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.core.annotation;
+
+import java.util.Map;
+
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.thrift.TException;
+
+import com.google.common.base.Supplier;
+
+import com.hotels.bdp.circustrain.api.CircusTrainException;
+import com.hotels.hcommon.hive.metastore.client.api.CloseableMetaStoreClient;
+
+public class BeekeeperHiveTableAnnotator implements HiveTableAnnotator {
```

Comment:
I think this should be made generic. CT should probably not know about beekeeper at all. The values of these properties should come from the yam configuration.